client/src/Components/ProductCard.tsx

```diff
@@ -37,8 +35,8 @@ export default function ProductCard(
             >
               -
             </button>
```

Comment:
Could we make these +/- buttons bordered too or more visible in any other way? Feels like they might be a bit lost because of small font size and a distance from quantity number, but maybe that's just to me😄

Reply:
I've made them white on hover, and they now grey out when they reach a limit (0 & 10)! Lemme know if more is needed :)